engine/apps/slack/scenarios/distribute_alerts.py

```diff
@@ -870,7 +934,7 @@ def process_signal(self, log_record: AlertGroupLogRecord) -> None:
         # Remove resolution note messages posted by OnCall bot
         for message in alert_group.resolution_note_slack_messages.filter(posted_by_bot=True):
             try:
-                self._slack_client.chat_delete(channel=message.slack_channel_id, ts=message.ts)
+                self._slack_client.chat_delete(channel=message.slack_channel_slack_id, ts=message.ts)
```

Comment:
this was a typo.

`ResolutionNoteSlackMessage` has a `slack_channel_slack_id` property which points to the Slack ID of the associated channel (ex. `C123FB4X`; `slack_channel_id` will actually point to the primary key ID of the associated `slack_channel` database record; ex. 14)